Sources/Protocols/MessagesLayoutDelegate.swift

```diff
@@ -125,22 +100,6 @@ public extension MessagesLayoutDelegate {
         return false
     }
 
-    // MARK: - Media Messages Defaults
-
-    func widthForMedia(message: MessageType, at indexPath: IndexPath, with maxWidth: CGFloat, in messagesCollectionView: MessagesCollectionView) -> CGFloat {
-        return maxWidth
-    }
-
-    func heightForMedia(message: MessageType, at indexPath: IndexPath, with maxWidth: CGFloat, in messagesCollectionView: MessagesCollectionView) -> CGFloat {
-        switch message.data {
-        case .photo(let image), .video(_, let image):
-            let boundingRect = CGRect(origin: .zero, size: CGSize(width: maxWidth, height: .greatestFiniteMagnitude))
-            return AVMakeRect(aspectRatio: image.size, insideRect: boundingRect).height
```

Comment:
@zhongwuzw The annoying thing is here is the default implementation. Now we're providing no default implementation. People are going to be confused where to get a width from I think 🤔

Reply:
Emm, I think it's ok, most of time, the image is loaded async, so user should provide size ahead.

Reply:
Not gonna lie, I'm still a little bothered that they have no frame of reference for the width. It just doesn't make sense.